HesoyamHospital/WebApplication/HospitalSurvey/SurveyMapper.cs

```diff
@@ -18,7 +18,7 @@ public static Survey SurveyDTOToSurvey(SurveyDTO dto)
             Section hygieneSection = new Section(dto.AnswerNine, dto.AnswerTen, dto.AnswerEleven, dto.AnswerTwelve);
             Section equipmentSection = new Section(dto.AnswerThirteen, dto.AnswerFourteen, dto.AnswerFifteen, dto.AnswerSixteen);
 
-            survey.Doctor = AppResources.getInstance().doctorService.GetByID(600);
+            survey.Doctor = AppResources.getInstance().doctorService.GetByID(700);
```

Comment:
Why is this value hardcoded?

Reply:
Also, if it is meant to be hardcoded, you should make it as variable instead of just writing its value as a parameter